scripts/lib/CIME/XML/machines.py

```diff
@@ -320,3 +320,30 @@ def print_values(self):
                 print("      pes/node       ",self.text(max_mpitasks_per_node))
             if max_tasks_per_node is not None:
                 print("      max_tasks/node ",self.text(max_tasks_per_node))
+
+    def return_values(self):
+        # return a dictionary of machine info
+        machines = self.get_children("machine")
+        mach_dict = dict()
+        logger.debug("Machines return values")
+        for machine in machines:
+            name = self.get(machine, "MACH")
+            
```

Comment:
remove this line

Reply:
which line do you want me to remove? line 331?

Reply:
yes